### ISSUE TITLE
Frontier Defense

### ISSUE THREAD

**Issue by RoyalBlue1** [R2Northstar/NorthstarMods#354]
These are the things that still need to be done to be able to release Frontier Defence split up in needed and nice to have
Needed:
- [ ] Cloak drone not cloaking
- [ ] Icons for enemies and friendly turrets on minimap
- [ ] Icon on Head of enemies (probably same as icon on minimap)
- [ ] Respawn with dropship
- [ ] Prevent Titan switching after wave 1
- [ ] Enemy Damage scaling
- [ ] Sniper titans
- [ ] Reaper tick deploy
- [ ] Damage scaling with difficulty
- [ ] Turret despawning after timeout and limit them per player
- [ ] wave data for at least one map

Nice to have:

- [ ] Combat and Support score
- [ ] Endgame Medals (most mvps, most damage, etc.)
- [ ] Round lost ui
- [ ] Remove clientside Harvester explosion when harvester out of render distance
- [ ] give money to late joiners

**Comment by x3Karma** [comment]
- [ ] Missing a delay before each wave where there's some dialogues of what's coming next.

**Comment by EpiicPenguin** [comment]
>     * [ ]  wave data for at least one map


non-programmer here, is this^ something that needs to be manual recorded with raw game play for enemy spawn numbers per wave?

I play a lot of frontier defense and am willing to put some time into logging data, just tell me what format you want the database in, and where to submit it. 

ps. love all you Northstar devs

**Comment by ASpoonPlaysGames** [comment]
Needed:

- [ ] Prevent drones from getting clumped up and stuck on each other
- [ ] Improve navmeshes to stop AI from clipping inside walls
- [ ] Make grunts use both anti-titan and normal weapons instead of just giving them anti-titan weapons on higher difficulties
- [ ] Titan selection screen UI
- [ ] Dropship grunt event (war games)
- [ ] Event functionality to allow for skipping events on lower difficulties

Nice to have:

- [ ] Make drones descend from the sky faster when they spawn
- [ ] Improve navmeshes to stop AI getting stuck
- [ ] Make SquadNav better, or make everything use SingleNav
- [ ] More robust mortar spectre pathing (probably needs jump points)
- [ ] Make drop pod AI actually come out of the drop pod door

**Comment by EnderBoy9217** [comment]
Colony and Relic were just added to the FD maps list FYI
